trustpinning/trustpin.go

```diff
@@ -37,6 +37,11 @@ func NewTrustPinChecker(trustPinConfig TrustPinConfig, gun data.GUN, firstBootst
 		t.pinnedCertIDs = pinnedCerts
 		return t.certsCheck, nil
 	}
+	var ok bool
+	t.pinnedCertIDs, ok = wildcardMatch(gun, trustPinConfig.Certs)
```

Comment:
nit: I think you can get away with `:=` here and avoiding the extra var, but only because this is the last use of `t.pinnedCertIDs`.  I'm generally of the opinion of the way you wrote this is more defensive though, so 👍